pkg/build/registry/rest.go

```diff
@@ -0,0 +1,52 @@
+package registry
+
+import (
+	"fmt"
```

Comment:
Since you use `fmt` only for errors and none actually requires formatting, use `errors` instead.